frontend/src/container/TopNav/DateTimeSelection/index.tsx

```diff
@@ -207,6 +215,10 @@ function DateTimeSelection({
 				setLocalStorageKey('startTime', startTimeMoment.toString());
 				setLocalStorageKey('endTime', endTimeMoment.toString());
 				updateLocalStorageForRoutes('custom');
+				urlQuery.set(QueryParams.startTime, startTimeMoment.toString());
+				urlQuery.set(QueryParams.endTime, endTimeMoment.toString());
+				const generatedUrl = `${location.pathname}?${urlQuery.toString()}`;
```

Comment:
can we use createQueryParams directly rather than adding directly

Reply:
done